lints/lint_old_root_ca_rsa_mod_less_than_2048_bits.go

```diff
@@ -21,14 +21,12 @@ func (l *rootCaModSize) Initialize() error {
 
 func (l *rootCaModSize) CheckApplies(c *x509.Certificate) bool {
 	issueDate := c.NotBefore
-	return (c.PublicKeyAlgorithm == x509.RSA && util.IsRootCA(c) && issueDate.Before(util.RsaDate2))
+	_, ok := c.PublicKey.(*rsa.PublicKey)
+	return (c.PublicKeyAlgorithm == x509.RSA && util.IsRootCA(c) && issueDate.Before(util.RsaDate2)) && ok
```

Comment:
I'm not sure if I agree with this. Not sure that I don't either though. The check _does_ apply, it's just a malformed certificate. That said, if another test checks for it, then, I could also see why we wouldn't say it's relevant. @kumarde how are we handling this in general?